apis/zalando.org/v1/types.go

```diff
@@ -168,6 +172,21 @@ type RouteGroupRouteSpec struct {
 	Methods []HTTPMethod `json:"methods,omitempty"`
 }
 
+// +k8s:deepcopy-gen=true
+type RouteGroupTLSSpec struct {
+	// TLS hosts specify the list of hosts included in the TLS secret.
+	// The values in this list must match the host name(s) used for 
+	// the RouteGroup in order to terminate TLS for the host(s).
+	// +kubebuilder:validation:MinItems=1
+	// +kubebuilder:validation:Required
+	Hosts []string `json:"hosts,omitempty"`
+
+	// SecretName is the name of the secret used to terminate TLS traffic.
+	// Secret should reside in the same namespace as the RouteGroup.
+	// +kubebuilder:validation:Required
```

Comment:
I think you just need to drop `omitempty` like we do for other string fields instead of using annotation.

Reply:
Looks better now, thanks for that.